index.html

```diff
@@ -367,7 +370,11 @@ <h4>Basic document structure</h4>
         <p>The following examples will demonstrate different uses in dubbing and audio description workflows.</p>
 
         <h4>Audio Description Examples</h4>
-        <p>When descriptions are added this becomes a <a>Pre-Recording Script</a>:</p>
+        <p>When descriptions are added this becomes a <a>Pre-Recording Script</a>.
+          Note that in this case, to reflect that most of the audio description content
+          transcribes the video image where there is no inherent language,
+          the <a>Text Language Source</a>, represented by the <a><code>daptm:langSrc</code></a> attribute,
+          is set to the empty string at the top level of the document:</p>
```

Comment:
Is it worth mentioning that it could be omitted because the default value is the empty string?

Reply:
Done in a1c80d4